Week3/homework/step2-4.js

```diff
@@ -1,10 +1,13 @@
 'use strict';
 
 function Dog() {
-  // add your code here
+  this.name = 'Golden';
+  this.color = 'gold';
+  this.numLegs = 4;
```

Comment:
This is a bit limiting. You've hard-coded Golden, gold, and 4, into the Dog constructor. That means any hound you create will always have these values.

The idea here is that Dog should take name, color, and numLegs as parameters - which you'd pass in when you create it. Like this;

const hound = new Dog('Golden', 'gold', 4);

Reply:
this.name=name
this.color=color
this.numLegs=Legs
const hound = new Dog('Golden', 'gold', 4);